src/apis/abusiveexperiencereport/README.md

```diff
@@ -7,20 +7,20 @@
 ## Installation
 
 ```sh
-$ npm install googleapis
+$ npm install @googleapis/abusiveexperiencereport
 ```
 
 ## Usage
 All documentation and usage information can be found on [GitHub](https://github.com/googleapis/google-api-nodejs-client).
+Information on classes can be found in [Googleapis Documentation](https://googleapis.dev/nodejs/googleapis/latest/abusiveexperiencereport/classes/Abusiveexperiencereport.html).
 
 ## Building a browser bundle
 
 This library can be used in a browser. To prepare a single file bundle, clone the
 [repository](https://github.com/googleapis/google-api-nodejs-client) and run
 
 ```sh
-$ cd src/apis/abusiveexperiencereport
-$ npm install
+$ npm install @googleapis/abusiveexperiencereport
```

Comment:
ugh, these instructions won't actually work :|    If you are trying to bundle for a browser now ....  I actually don't know what to tell you.  Before this worked because people were responsible for building their own bundles and then copying them over.  Now that we ship a sub module, we either need to:
1. Keep these instructions the same OR
2. Start to run `npm run webpack` as part of our builds, and publish something in the `browser` field in `package.json`


This area isn't my strength uhhh...  @bcoe?

Reply:
I just revisited yargs' webpack config, so was playing a bit with it on the weekend:

https://github.com/yargs/yargs/pull/1900

What if we remove this section entirely for now, but open a tracking issue to add a general `docs/webpack.md` guide at the top level? This could include step by step instructions for webpacking the individual modules (which I think would work similarly to yargs).

Reply:
+1 to removing the webpack instructions all up.  

For future tracking though - my proposal (I think) is that we do the `npm run webpack` as part of `npm run compile`, and package the results in the npm module.  I'm not 100% sure of if/how `webpack` supports the "browser" field "spec", but if it does we should consider using that instead.

Assume users won't have any idea how to use webpack.

Reply:
Opened an issue:

https://github.com/googleapis/google-api-nodejs-client/issues/2584

@sofisl tldr; let's remove the webpack section from the individual READMEs.

Reply:
Done!